frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -215,34 +347,131 @@ const ManagePolicyPage = (): JSX.Element => {
           )}
         </div>
         <div className={`${baseClass}__description`}>
-          <p>Policy queries report which hosts are compliant.</p>
-        </div>
-        {policies && policies.length !== 0 && !isLoadingError && (
-          <InfoBanner className={`${baseClass}__sandbox-info`}>
+          {isPremiumTier && !!selectedTeamId && (
             <p>
-              Your policies are checked every <b>{updateInterval.trim()}</b>.
-              Check out the Fleet documentation on{" "}
-              <a href={DOCS_LINK} target="_blank" rel="noreferrer">
-                <b>how to edit this frequency</b>
-              </a>
+              Add additional policies for <b>all hosts assigned to this team</b>
               .
             </p>
-          </InfoBanner>
-        )}
-        <div>
-          {renderTable(
-            policies,
-            isLoading,
-            isLoadingError,
-            onRemovePoliciesClick,
-            toggleAddPolicyModal
           )}
+          {isFreeTier ||
+            (isPremiumTier && !selectedTeamId && selectedTeamId !== null && (
```

Comment:
If booleans are this long inside the JSX, let's create variables for them. Before the return statement (~ line 316), you could do something like:
```typescript
const shouldUpsell = isFreeTier || (isPremiumTier && !selectedTeamId && selectedTeamId !== null);
```

Then, use that boolean in the JSX. This way, devs can understand the logic better at first glance when we give it meaningful name.

@RachelElysia including you on this note.